js/script.js

```diff
@@ -14,13 +14,15 @@ var params = {
 	playerName: 'Player',
 	progress: [],
 	buttonSet: document.querySelectorAll('.player-move')
-}
+};
 
 // Add event listeners to buttons.
-for (var i = 0; i < params.buttonSet.length; i++) {
-	params.buttonSet[i].addEventListener('click', playerMove);
-	params.buttonSet[i].value = params.buttonSet[i].getAttribute('data-move');
-}
+(function initializeListeners() {
```

Comment:
jak robimy IIFE to nie ma sensu jej nazywać - bo i tak jej potem nie używasz, natomiast samo IIFE jest tutaj niepotrzebne - zrób jedną, normalną funkcję `initializeListeners`, która zainicjalizuje i te listenery i te z dołu.

samo wywołanie `initializeListeners()` może znajdować się gdzieś na górze, po deklaracji `params`